eiffel-vocabulary/EiffelServiceDeployedEvent.md

```diff
@@ -45,30 +45,162 @@ __Type:__ String
-__Multiple allowed:__ No  
-__Description:__ Identifies the environment in which an event occurred, e.g. in which environment an artifact was built or an issue was verified.
+## Legal Link Types 
+
+### ARTIFACT_DEPLOYED
+__Required:__ No  
+__Legal targets:__ [EiffelArtifactDeployedEvent](./EiffelArtifactDeployedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies what artifacts that have been deployed as a part of the service
 
 ### CAUSE
+__Required:__ No  
 __Legal targets:__ Any  
 __Multiple allowed:__ Yes  
 __Description:__ Identifies a cause of the event occurring. SHOULD not be used in conjunction with __CONTEXT__: individual events providing __CAUSE__ within a larger context gives rise to ambiguity. It is instead recommended to let the root event of the context declare __CAUSE__.  
 
 ### CONTEXT
-__Legal targets:__ [EiffelActivityTriggeredEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelActivityTriggeredEvent.md), 
+__Required:__ Yes  
+__Legal targets:__ [EiffelServiceAllocated](./EiffelServiceAllocated.md)  
 __Multiple allowed:__ No  
-__Description:__ Identifies the activity of which the event constitutes a part. SHOULD not be used in conjunction with __CAUSE__, see above. Note that multiple layers may be modeled using __CONTEXT__, e.g. an activity being part of another activity.
+__Description:__ Identifies the service that artifacts was deployed at
 
-### ACTIVITY_EXECUTION
-__Legal targets:__ [EiffelActivityTriggeredEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelActivityTriggeredEvent.md)  
+### ENVIRONMENT
+__Legal targets:__ [EiffelEnvironmentDefinedEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md)  
 __Multiple allowed:__ No  
-__Description:__ Declares the activity execution the event relates to. In other words, [EiffelActivityTriggeredEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelActivityTriggeredEvent.md) acts as a handle for the activity execution. This differs from __CONTEXT__. In __ACTIVITY_EXECUTION__ the source carries information pertaining to the target (i.e. the activity started, finished or was canceled). In __CONTEXT__, on the other hand, the source constitutes a subset of the target.
+__Description:__ Identifies the environment in which an event occurred, e.g. in which environment an artifact was built or an issue was verified.
+
+### FLOW_CONTEXT
+__Required:__ No  
+__Legal targets:__ [EiffelFlowContextDefinedEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelFlowContextDefinedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies the flow context of the event: which is the continuous integration and delivery flow in which this occurred - e.g. which product, project, track or version this is applicable to.
+
+
+### SERVICE
+__Required:__ Yes  
```

Comment:
Required could be NO.

If we do not have any orchestrator tool and fixed hardware, then we will not have a service allocated event.